questionnaires/esm28q/esm28q_armt_nl.json

```diff
@@ -681,31 +681,31 @@
                 "label": "Reizen "
             },
             {
-                "code": "12",
+                "code": "6",
```

Comment:
These are not live yet, right? I that case changes to code should be fine.

Reply:
Yup they're not yet released to participants. I think these are corrections to the codes cause they didn't match the English version.